src/osemgrep/cli_install/Install_CLI.ml

```diff
@@ -72,6 +73,15 @@ let o_update : bool Term.t =
   let doc = "Update any existing workflow and secrets (absent=false)." in
   H.negatable_flag [ "update" ] ~neg_options:[ "no-update" ] ~default:false ~doc
 
+(* NOTE: Should only be required when previous attempts only partially succeeded *)
+let o_dry_run : bool Term.t =
+  let doc =
+    "Simulate outputs without actually running any system commands \
+     (absent=false)."
+  in
+  H.negatable_flag [ "dry-run" ] ~neg_options:[ "no-dry-run" ] ~default:false
```

Comment:
I don't think you need the negatable option here.
just "dry-run" should be enough, nobody is gonna ever used --no-dry-run